### PR TITLE
Logo Changes

### LINKED ISSUE #88: Header - Logo has no link to home
On current site user can click the logo to go "home"
Screenshot 2024-08-01 at 12 30 16 PM

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #88 

Test URLs:
- Before: https://main--shredit--stericycle.aem.live/drafts/piyush/en-us
- After: 
  - https://hover-change--shredit--stericycle.aem.live/drafts/piyush/en-us
  - https://hover-change--stericycle-shared--aemsites.aem.live/drafts/piyush/en-us

### CODE REVIEW

**Comment by jindaliiita** [comment]
Due to some cache issue getting 404 for the branch hover-change

File: blocks/header/header.js [hunk 1]
```diff
@@ -332,7 +332,8 @@ export default async function decorate(block) {
   const locale = window.location.pathname.split('/')[1] || 'en-us'; // default to us-en if no locale in path
   // load nav as fragment
   const navMeta = getMetadata('nav');
-  const navPath = navMeta ? new URL(navMeta, window.location).pathname : `/${locale}/nav`;
+  // todo remove below comment, only for testing purpose
+  const navPath = ('/drafts/piyush/nav') || (navMeta ? new URL(navMeta, window.location).pathname : `/${locale}/nav`);
```
Comment (kronnox) [inline blocks/header/header.js:336]: I don't think you meant to leave that hardcoded ;)

**Review by kronnox — DISMISSED** [review]

File: blocks/header/header.js (hunk 1 above)
Comment (jindaliiita) [reply blocks/header/header.js:336]: Its only for testing purpose :)

**Comment by jindaliiita** [comment]
@kronnox Let me know if i can merge this

**Comment by kronnox** [comment]
Sorry didn't see the ping. I don't see any issues with merging this

**Review by dkuntze — APPROVED** [review]

**Pull request merged**